mockfirestore/transaction.py

```diff
@@ -0,0 +1,119 @@
+from functools import partial
+import random
+from typing import Iterable, Callable
+from string import ascii_letters, digits
+from .main import MockFirestore, DocumentReference, DocumentSnapshot, Query, Timestamp
+
+MAX_ATTEMPTS = 5
+ALPHABET = list(ascii_letters) + list(digits)
+_MISSING_ID_TEMPLATE = "The transaction has no transaction ID, so it cannot be {}."
+_CANT_BEGIN = "The transaction has already begun. Current transaction ID: {!r}."
+_CANT_ROLLBACK = _MISSING_ID_TEMPLATE.format("rolled back")
+_CANT_COMMIT = _MISSING_ID_TEMPLATE.format("committed")
+
+
+class MockWriteResult:
```

Comment:
For consistency with the other classes this should just be called `WriteResult`

Reply:
Okay, will fix.